hacks/bump-jenkins-version/bump-jenkins.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+#https://updates.jenkins-ci.org/download/war/2.73.1/jenkins.war
+
+JENKINS_DIST_GIT=jenkins
+
+usage() {
+    echo >&2
+    echo "Usage `basename $0` <jenkins-war-version> <branch>" >&2
+    echo >&2
+    echo " Example: ./bump-jenkins.sh 2.73.1 rhaos-3.7-rhel-7" >&2
+    exit 1
+}
+
+# clone jenkins distgit
+# switch to latest branch 
+setup_dist_git() {
+  container="jenkins"
```

Comment:
`container` isn't accurate. And can't this use $JENKINS_DIST_GIT ?

Reply:
✔️